README.md

```diff
@@ -1,53 +1,136 @@
+Due to large file size use md5sum to verify the hash on 
+each platform like so:
 
-# Do this to update V8 and its dependencies. The reset command is required
-# as the current patches are valid as of the specified commit.
-git pull
-git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
-gclient sync
-rm -rf test tools third_party
 ```
-From another platform (e.g. Linux), perform the same steps above (but without
-the rm command), then transfer the directories test, tools and third_party to
-your z/OS machine.
+$ md5sum x.tar
+```
+
+
+#### z/OS® UNIX System Services (z/OS UNIX) platform setup
+
+This is the workflow on the z/OS® UNIX System Services (z/OS UNIX) platform.
+
+It assumes you are using z/OS Open Tools framework and the associated install
+of depot_toolsport.  Follow the workflow described [here](https://github.com/ZOSOpenTools/depot_toolsport?tab=readme-ov-file#developer-notes) to ensure you have deeopot tools properly installed 
+and configured.
+
+##### Do this once
+
+This uses zopen depot tools port to obtain a copy of the v8 source
+according to the v8 workflow.
 
-Create an archive using `tar` (on the non-z/OS platform):
 ```
-cd v8
-tar -cvf x.tar test tools third_party
+$ mkdir $HOME/zopen/dev/v8base
+$ cd $HOME/zopen/dev/v8base
+$ fetch v8
 ```
-Transfer x.tar to the z/OS machine using your preferred method (e.g. `sftp`).
 
-Then on the z/OS machine:
+Obtain a valid git repo.
+
 ```
-cd v8
-# extract the archive:
-tar -xvf x.tar
-# tag all files in those directories as ASCII:
-chtag -tc819 -R test tools third_party
-rm x.tar
+$ cd $HOME/zopen/dev/v8base/v8
+$ git checkout main
+```
+
+##### Do this afterwards and everytime you need to sync
+
+Once you have done the intial setup above, these are the
+steps to sync the code to the specified revision and apply the corresponding
+files from the Linux AMD64 platform for the files normally 
+acquired via `cipd`.
+
+The reset command is required as the current patches are valid as of the specified commit.
+
+```
+$ cd $HOME/zopen/dev/v8base/v8
+$ git pull
+$ git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
+$ gclient sync -D
+$ rm -rf test tools third_party
+```
+
+At this point the file system is ready to be suplemented with the files
+in the tar file from the non z/OS UNIX platform.
+
+```
+$ cd $HOME/zopen/dev/v8base/v8
+$ tar -xvf ~/zopen/x.tar
+```
+
+Then make sure all the files are properly tagged as ASCII:
+
+```
+$ cd $HOME/zopen/dev/v8base/v8
+$ chtag -tc819 -R test tools third_party
+```
+
+Optionally remove the tar file as its not longer needed.  It will 
+need to be replaced if sync is performed, however keep in mind
+this will also require the workflow on the non z/OS UNIX system 
+to recreate it.
+
+```
+$ rm x.tar
 ```
 
 Ensure that the head commit on both z/OS and the other platform are the same; if
 not then `git reset <commit> --hard` on z/OS to match the commit on the other
 platform.
 
```

Comment:
Please remove this paragraph. The other steps ensure that that's the case; a user may blindly run the git reset to make sure, but that then requires a gclient sync, rm -rf test .... and tar -xvf ...

Reply:
done.